.github/workflows/teleport-create-role.yaml

```diff
@@ -2,10 +2,17 @@ name: Create teleport role
 on:
   workflow_dispatch:
       inputs:
+        user_manage:
+          description: "User manage teleport cluster"
+          type: string
+        password_user_manage:
+          description: "Password user manage teleport cluster"
```

Comment:
user_manage -> teleport_admin_user
password_user_manage -> teleport_admin_password 
"manage" is a verb (động từ) ; ) 
same for the description